x-pack/legacy/plugins/apm/public/components/app/ErrorGroupOverview/index.tsx

```diff
@@ -53,42 +54,62 @@ const ErrorGroupOverview: React.SFC = () => {
     }
   }, [serviceName, start, end, sortField, sortDirection, uiFilters]);
 
-  useTrackPageview({ app: 'apm', path: 'error_group_overview' });
+  useTrackPageview({
+    app: 'apm',
+    path: 'error_group_overview'
+  });
   useTrackPageview({ app: 'apm', path: 'error_group_overview', delay: 15000 });
 
+  const localUIFiltersConfig = useMemo(() => {
+    const config: React.ComponentProps<typeof LocalUIFilters> = {
+      filterNames: ['transactionResult', 'host', 'containerId', 'podId'],
+      params: {
+        serviceName
```

Comment:
How is this type checked so that we ensure the correct params are passed?

Reply:
Hmm, good question. We probably need to abstract the part in routes/ui_filters that maps endpoints to query parameters, and then use that in LocalUIFilters or here as well. Let me think about it for a bit how that would look.